membership-attribute-service/app/services/AttributesFromZuora.scala

```diff
@@ -136,10 +150,16 @@ class AttributesFromZuora(implicit val executionContext: ExecutionContext, syste
     subscriptionsForAccountId: AccountId => SubPlanReads[AnyPlan] => Future[Disjunction[String, List[Subscription[AnyPlan]]]]
   ): Future[Disjunction[String, List[AccountWithSubscriptions]]] = {
 
-    def accountWithSubscriptions(account: AccountObject)(implicit reads: SubPlanReads[AnyPlan]): Future[Disjunction[String, AccountWithSubscriptions]] =
-      EitherT(subscriptionsForAccountId(account.Id)(anyPlanReads))
+    def isNotDigipackGiftSub(subscription: Subscription[AnyPlan]) = subscription.asDigipack.isEmpty || subscription.readerType != Gift
+
+    def accountWithSubscriptions(account: AccountObject)(implicit reads: SubPlanReads[AnyPlan]): Future[Disjunction[String, AccountWithSubscriptions]] = {
+      val nonDigipackGiftSubs = subscriptionsForAccountId(account.Id)(anyPlanReads).map(_.map(
```

Comment:
minor - could have wrapped in the EitherT before the `map.map` which would then only be a single `.map`